### PR TITLE
React Percy Docs

### PR DESCRIPTION
**Pull request by thomashuston**
Summary
--
Made a first pass at documentation for using `react-percy`. We'll probably want to tighten these up a bit and expand them, but hopefully this will be helpful for the initial beta testers!

### CODE REVIEW

File: docs/clients/javascript/react.md [hunk 1]
```diff
@@ -15,3 +15,299 @@ If you'd like to test your React app today, check out [Percy Anywhere](/docs/cli
...
+```javascript
+const Button = require('./Button');
+const React = require('react');
+
+percySnapshot('Button', function() {
+  return <Button>My Button</Button>
+});
+```
+
+Add the following section to your `package.json`, replacing `<path/to/webpack.config.js>` with the path of the Webpack config file in your project:
```
Comment (timhaines) [inline docs/clients/javascript/react.md]: Maybe worth mentioning the --config option isn't required for create-react-app apps.  Or is that just for un-ejected CRA apps?

File: docs/clients/javascript/react.md [hunk 2]
```diff
@@ -15,3 +15,299 @@ If you'd like to test your React app today, check out [Percy Anywhere](/docs/cli
...
+percySnapshot('PlayerInfo: expanded', function() {
+  return <PlayerInfo {...props} country="USA" sport="Tennis" />;
+});
+```
+
+### Grouping Snapshots
+
+You may also group snapshots together using a `suite` block.
+
+When they are inside a `suite` block, `beforeEach` blocks only apply to the snapshots within that `suite` block.
```
Comment (timhaines) [inline docs/clients/javascript/react.md]: This confused me a little, as I expected to see `beforeEach` in the code sample below.

File: docs/clients/javascript/react.md (hunk 1 above)
Comment (thomashuston) [reply docs/clients/javascript/react.md]: Added a note below for `create-react-app` users

File: docs/clients/javascript/react.md (hunk 2 above)
Comment (thomashuston) [reply docs/clients/javascript/react.md]: Good point. I moved this down and added a corresponding example.

File: docs/clients/javascript/react.md [hunk 3]
```diff
@@ -1,17 +1,470 @@
...
+This will generate 320px and 768px wide snapshots of "Button: primary", "Button: secondary", "Button: submit: enabled", and "Button: submit: disabled".
+
+## Troubleshooting
+
+### Percy is rendering a blank white page
+
+If you're seeing blank white snapshots in Percy, it usually means an error occurred client-side when loading your snapshot. To debug this, you can run `react-percy` locally with the `--debug` flag so you can load snapshots in your local browser.
+
+```sh
+$ react-percy --config ./path/to/webpack.config.js --debug
```
Comment (timhaines) [inline docs/clients/javascript/react.md]: @thomashuston If I understand correctly, this line should now read:
```sh
$ react-percy --debug
```

**Pull request merged**